### PR TITLE
Make loripsum mockable

### PR DESCRIPTION
**Pull request by rossriley**

### CODE REVIEW

**Comment by GawainLynch** [comment]
:+1:

File: src/Provider/PrefillServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+use Bolt\Storage\Prefill;
+
+class PrefillServiceProvider implements ServiceProviderInterface
+{
+    public function register(Application $app)
+    {
+        $app['prefill'] = $app->share(
```
Comment (GawainLynch) [inline src/Provider/PrefillServiceProvider.php]: I know this to some extent belongs in our discussion about the 2.2 refactor, but while adding a `Bolt\Storage` namespace, should we also namespace the services, i.e. `$app['storage.prefill']`?

Comment (CarsonF) [inline src/Provider/PrefillServiceProvider.php]: I agree

**Comment by rossriley** [comment]
Just hold off on merging this one for a bit.. need to add a test for Prefill.

File: src/Provider/PrefillServiceProvider.php (hunk 1 above)
Comment (GawainLynch) [inline src/Provider/PrefillServiceProvider.php]: Consensus on IRC was this is a 2.2-dev discussion. :nut_and_bolt:

**Comment by rossriley** [comment]
Ok all good now.

**Comment by GawainLynch** [comment]
:cake:

**Pull request merged**